modin/backends/pandas/query_compiler.py

```diff
@@ -53,6 +53,8 @@
     groupby_reduce_functions,
 )
 
+from modin.backends.base.query_compiler import _get_axis as _get_axis_
```

Comment:
Propose to move import of `_get_axis` to line
```python
from modin.backends.base.query_compiler import BaseQueryCompiler, _get_axis
```

Reply:
I think we can get rid of `_get_axis` import at all if we would inherit the `index` property from the `BaseQueryCompiler` (since it doesn't defaulting to pandas anymore). `PandasQueryCompiler` overwrites `index` prop at [this line](https://github.com/modin-project/modin/blob/cf259f1cd38c70e246fd5e3ea904b675a5d10bd9/modin/backends/pandas/query_compiler.py#L266), this one should be removed.

Reply:
I'm wrong, we can't do that because the `index` property also contains `_set_axis(0)` as a setter, which is different in base and pandas query compilers